internal/core/core.go

```diff
@@ -539,3 +539,21 @@ func levenshteinWithMax(s1 []rune, s2 []rune, maxValue float64) float64 {
 func Identity[T any](t T) T {
 	return t
 }
+
+func CheckEachDefined[S any](s []*S, msg string) []*S {
```

Comment:
Are these functions used? The codecov extension shows them as uncovered.

Reply:
Not yet. They're used in a function in `completions.ts` that is not currently exercised by the tests, because I haven't added enough tests yet.